scripts/tick-cluster.js

```diff
@@ -429,7 +487,7 @@ function killProc(count) {
         var proc = procs[rand];
         if (proc.killed === null && proc.suspended === null) {
             logMsg(proc.port, color.green('pid ' + proc.pid) + color.red(' randomly selected for death'));
-            process.kill(proc.proc.pid, 'SIGTERM');
+            process.kill(proc.proc.pid, 'SIGKILL');
```

Comment:
I can't find it right now but it terminates the process. You can catch the signal with the os package to shut down gracefully.

We want to simulate a node failing here so I thought SIGKILL would be more appropriate.